control/cli.py

```diff
@@ -36,11 +44,40 @@ def __init__(self):
             prog="python3 -m control.cli",
             description="CLI to manage NVMe gateways")
         self.parser.add_argument(
-            "-c",
-            "--config",
-            default="ceph-nvmeof.conf",
+            "--server-address",
+            default="ceph-nvmeof_ceph-nvmeof_1",
+            type=str,
+            help="Server address",
+        )
+        self.parser.add_argument(
+            "--server-port",
+            default=5500,
+            type=int,
+            help="Server port",
+        )
+        self.parser.add_argument(
+            "--server-auth",
+            default=False,
+            type=str_to_bool,
+            help="Enable server authentication"
+        )
```

Comment:
In general, flags (boolean options) are simply defined without `type` and with `action='store_true'`. If you really want to expose both an on and an off flag (do we really?), there's this [`BooleanOptionalAction`](https://docs.python.org/3/library/argparse.html#action:~:text=The%20BooleanOptionalAction%20is%20available%20in%20argparse%20and%20adds%20support%20for%20boolean%20actions%20such%20as%20%2D%2Dfoo%20and%20%2D%2Dno%2Dfoo%3A), which would support both `--server-auth` and `--no-server-auth`.

That said, from a usability standpoint, given that the `--server-auth` option is not enough by itself to enable this, I'd omit it and enable server-auth as long as both `--client-key` and `--client-cert` pair is provided.